packages/event-handler/src/rest/utils.ts

```diff
@@ -43,3 +44,38 @@ export function validatePathPattern(path: Path): ValidationResult {
     issues,
   };
 }
+
+type ProcessedParams = {
+  raw: Record<string, string>;
+  processed: Record<string, any>;
+  route: Route;
+};
+
+export function processParams(
```

Comment:
As mentioned in the body of the PR, I think both these utilities can probably llive in the registry. Thoughts?

Reply:
I don't at this stage - but my rule of thumb recommendation would be to start everything as `#private`, move to `protected` if we think customers might want to extend it (or they ask us), then leave `public` only things we are 100% intending them to use.